tests/functional/test_utilities_parameters.py

```diff
@@ -1639,14 +1639,22 @@ def test_appconf_provider_get_configuration_json_content_type(mock_name, config)
     encoded_message = json.dumps(mock_body_json).encode("utf-8")
     mock_value = StreamingBody(BytesIO(encoded_message), len(encoded_message))
 
-    # Stub the boto3 client
     stubber = stub.Stubber(provider.client)
-    response = {"Content": mock_value, "ConfigurationVersion": "1", "ContentType": "application/json"}
-    stubber.add_response("get_configuration", response)
+    response_start = {"InitialConfigurationToken": "initial_token"}
+    stubber.add_response("start_configuration_session", response_start)
```

Comment:
```suggestion
    response_start_config_session = {"InitialConfigurationToken": "initial_token"}
    stubber.add_response("start_configuration_session", response_start_config_session)
```

easier to read which response belongs to which API